frontend/src/sass/pages/_home.scss

```diff
@@ -25,7 +25,7 @@
     color: $colr-txt-accent;
 }
 
-.pnl-causes, .pnl-marketers, .pnl-marketing, .pnl-signup, .pnl-contact-us, .pnl-talkabout, .pnl-hero  {
+.pnl-marketing, .pnl-marketers, .pnl-signup, .pnl-contact-us, .pnl-talkabout, .pnl-hero, .pnl-causes  {
```

Comment:
the order of comma separated selectors is not important no. It will apply to all of them regardless of order.